apps/cli/src/node/docker-compose-dev.yaml

```diff
@@ -83,75 +83,8 @@ services:
             - 4000
             - 5004
             - 5005
-        environment:
-            # s6-overlay
-            S6_STAGE2_HOOK: "/etc/s6-overlay/scripts/stage2-hook.sh"
-            S6_VERBOSITY: "${S6_VERBOSITY:-2}"
-            # global
-            RUST_LOG: "${RUST_LOG:-info}"
-            ## shared
-            ### redis
-            REDIS_ENDPOINT: redis://redis:6379
-            ### contract-address-file
-            DAPP_CONTRACT_ADDRESS_FILE: /usr/share/sunodo/dapp.json
-            ### session-id
-            SESSION_ID: default_session_id
-            ### chain-id
-            CHAIN_ID: 31337
-            TX_CHAIN_ID: 31337
-            ### server-manager-endpoint
-            SERVER_MANAGER_ENDPOINT: http://localhost:5001
-            SERVER_MANAGER_ADDRESS: localhost:5001
-            SERVER_MANAGER_LOG_LEVEL: "${SERVER_MANAGER_LOG_LEVEL:-info}"
-            REMOTE_CARTESI_MACHINE_LOG_LEVEL: "${REMOTE_CARTESI_MACHINE_LOG_LEVEL:-info}"
-
-            ### database
-            POSTGRES_ENDPOINT: "${POSTGRES_ENDPOINT:-postgres://postgres:password@database:5432/postgres}"
-
-            # dispatcher
-            ## uses redis
-            ## uses chain-id (TX_CHAIN_ID acctually)
-            AUTH_MNEMONIC: "test test test test test test test test test test test junk"
-            RD_DAPP_DEPLOYMENT_FILE: /usr/share/sunodo/dapp.json
-            RD_ROLLUPS_DEPLOYMENT_FILE: /usr/share/sunodo/localhost.json
-            RD_EPOCH_DURATION: "${EPOCH_DURATION:-86400}"
-            SC_GRPC_ENDPOINT: http://localhost:50051
-            SC_DEFAULT_CONFIRMATIONS: 1
-            TX_PROVIDER_HTTP_ENDPOINT: http://anvil:8545
-            TX_CHAIN_IS_LEGACY: ${TX_LEGACY:-false}
-            TX_DEFAULT_CONFIRMATIONS: 2
-
-            # state-server
-            SF_GENESIS_BLOCK: 1
-            SF_SAFETY_MARGIN: 1
-            BH_HTTP_ENDPOINT: http://anvil:8545
-            BH_WS_ENDPOINT: ws://anvil:8545
-            BH_BLOCK_TIMEOUT: "${BLOCK_TIMEOUT:-8}"
-
-            # advance-runner
-            ## uses redis
-            ## uses contract-address-file
-            ## uses session-id
-            ## uses chain-id
-            ## uses server-manager-endpoint
-            PROVIDER_HTTP_ENDPOINT: http://anvil:8545
-            SNAPSHOT_DIR: "/var/opt/cartesi/machine-snapshots"
-            SNAPSHOT_LATEST: "/var/opt/cartesi/machine-snapshots/latest"
-
-            # inspect-server
-            ## uses session-id
-            ## uses server-manager-endpoint (acctually uses SERVER_MANAGER_ADDRESS)
-            INSPECT_SERVER_ADDRESS: 0.0.0.0:5005
-
-            # indexer
-            ## uses redis
-            ## uses contract-address-file
-            ## uses chain-id
-
-            # graphql-server
-            ## uses postgres-config
-            GRAPHQL_HOST: 0.0.0.0
-            GRAPHQL_PORT: 4000
+        env_file:
+            - ${SUNODO_BIN_PATH}/node/default.env
```

Comment:
We may get similar error on macos according to the issue #245

Reply:
Can you test this? I don't have a maOS.

Reply:
Me either !
I think @tuler can help.

Reply:
I think this is not the same case from #245 , since it's not a volume mapping.

Reply:
If I remove the docker volume mapping permission it does not work properly.

Strangely it does not work in non-verbose mode. But works in verbose mode.

Reply:
But this PR makes no changes to volumes. :thinking:

Reply:
Yeah, it may be something not introduced by this PR.
Topic for #267 and the removal of volume mapping in sunodo dir.